### PR TITLE
Fixes #99 : Adding Links

### LINKED ISSUE #99: Adding Links to Navbar and Footer, Remmoving Question Mark Links

### PR DESCRIPTION
**Pull request by saarthakchats**
Fixes #99 
 
> @0x48piraj collaborated on my repository, sent a PR :+1:

### CODE REVIEW

**Comment by saarthakchats** [comment]
https://saarthakchats.github.io/my-n-a-m-e-i-s2012.fossasia.org/

File: _includes/footer.html [hunk 1]
```diff
@@ -37,7 +37,7 @@
                                 <li><a target="_self" href="https://www.youtube.com/fossasiaorg"><i class="icon social_youtube icon-large"></i></a></li>
                                 <li><a target="_self" href="https://www.flickr.com/photos/fossasia"><i class="icon social_flickr icon-large"></i></a></li>
                                 <li><a target="_self" href="https://www.instagram.com/explore/tags/fossasia/"><i class="icon social_instagram"></i></a></li>
-                                <li><a target="_self" href="https://github.com/fossasia"><i class="fa fa-github fa-lg"></i></a></li>
+                                <li><a href="https://plus.google.com/108920596016838318216" target="default"><i class="icon social_googleplus"></i></a></li>
                             </ul>
```
Comment (abishekvashok) [inline _includes/footer.html]: Why do you remove the github reference?

File: _includes/header.html [hunk 2]
```diff
@@ -72,7 +72,8 @@
                                     <li><a href="https://www.instagram.com/fossasia" target="default"><i class="icon social_instagram"></i></a></li>
                                     <li><a target="_self" href="https://www.flickr.com/photos/fossasia"><i class="icon social_flickr"></i></a></li>
                                     <li><a href="https://plus.google.com/108920596016838318216" target="default"><i class="icon social_googleplus"></i></a></li>
-                                    <li><a href="https://github.com/fossasia" target="_self" ><i class="fa fa-github fa-lg"></i></a></li>
```
Comment (abishekvashok) [inline _includes/header.html:75]: Here as well, why was it removed?

File: _includes/footer.html [hunk 3]
```diff
@@ -37,7 +37,7 @@
                                 <li><a target="_self" href="https://www.youtube.com/fossasiaorg"><i class="icon social_youtube icon-large"></i></a></li>
                                 <li><a target="_self" href="https://www.flickr.com/photos/fossasia"><i class="icon social_flickr icon-large"></i></a></li>
                                 <li><a target="_self" href="https://www.instagram.com/explore/tags/fossasia/"><i class="icon social_instagram"></i></a></li>
-                                <li><a target="_self" href="https://github.com/fossasia"><i class="fa fa-github fa-lg"></i></a></li>
```
Comment (naveentherocker) [inline _includes/footer.html]: Why did you remove this?

File: _includes/header.html (hunk 2 above)
Comment (naveentherocker) [inline _includes/header.html]: This too

**Comment by saarthakchats** [comment]
I'll add them back

**Comment by NaviRocker** [comment]
@saarthakchats If possible, Try to squash your commits too.

File: _includes/footer.html [hunk 4]
```diff
@@ -37,6 +37,7 @@
                                 <li><a target="_self" href="https://www.youtube.com/fossasiaorg"><i class="icon social_youtube icon-large"></i></a></li>
                                 <li><a target="_self" href="https://www.flickr.com/photos/fossasia"><i class="icon social_flickr icon-large"></i></a></li>
                                 <li><a target="_self" href="https://www.instagram.com/explore/tags/fossasia/"><i class="icon social_instagram"></i></a></li>
+                                <li><a href="https://plus.google.com/108920596016838318216" target="default"><i class="icon social_googleplus"></i></a></li>
```
Comment (NaviRocker) [inline _includes/footer.html:40]: For all the Other Links, Target is `_self`. Your one, it 's `default`. Better to maintain every social link as same.

**Review by NaviRocker — CHANGES_REQUESTED** [review]

**Comment by saarthakchats** [comment]
Squashed

@naveentherocker @abishekvashok @harshit98 @vaibhavsingh97 
Please approve

**Review by NaviRocker — APPROVED** [review]
LGTM

**Review by harshit98 — APPROVED** [review]
